### PR TITLE
Fix photon eta cut for trigger efficiency 

### PR DESCRIPTION
**Pull request by caleb-james-smith**
- Fix photon eta cut for trigger efficiency 
- Change tabs to spaces

### CODE REVIEW

File: python/modules/Stop0l_trigger.py [hunk 1]
```diff
@@ -183,108 +183,105 @@ def analyze(self, event):
...
-                        #        photon_mid.append(photon)
-			if self.era == "2016":
-				if photon.cutBased >=1: photon_loose.append(photon)
-				if photon.cutBased >=2: photon_mid.append(photon)
-			else:
-				if bool(photon.cutBasedBitmap & 1): photon_loose.append(photon)
-				if bool(photon.cutBasedBitmap & 2): photon_mid.append(photon)
+                # CMSSW Reference for eta cuts: https://github.com/cms-sw/cmssw/blob/02d4198c0b6615287fd88e9a8ff650aea994412e/DQM/Physics/python/singleTopDQM_miniAOD_cfi.py#L47
+                abeta = math.fabs(photon.eta)
+                if (abeta > 1.4442 and abeta < 1.5660) or (abeta > 2.5):
```
Comment (zhenbinwu) [inline python/modules/Stop0l_trigger.py]: Why do you change this? This selects photon in the gap or > 2.5 only!

Comment (zhenbinwu) [reply python/modules/Stop0l_trigger.py]: @wang-hui This is the selection you are using?

Comment (wang-hui) [reply python/modules/Stop0l_trigger.py]: Hi Caleb, I think you are selecting the wrong region.
Could you please change to
if (abs(photon.eta) < 1.4442 or (1.566 < abs(photon.eta) and abs(photon.eta) < 2.5)):

Comment (caleb-james-smith) [reply python/modules/Stop0l_trigger.py]: Ah sorry... my mistake. I used copy paste from our Photon Stop0l selection which actually vetos photons passing "(abeta > 1.4442 and abeta < 1.5660) or (abeta > 2.5)".

Comment (caleb-james-smith) [reply python/modules/Stop0l_trigger.py]: It is probably safer to just use the photon Stop0l flag that is already calculated... but I will fix the if statement for now.

Comment (caleb-james-smith) [reply python/modules/Stop0l_trigger.py]: Here is the Stop0l eta cut I used, but with opposite logic. I will what I put for the trigger selection. 
https://github.com/susy2015/NanoSUSY-tools/blob/dev_v3/python/modules/Stop0lObjectsProducer.py#L150-L151

Comment (caleb-james-smith) [reply python/modules/Stop0l_trigger.py]: What I wanted to change was 1.442 to 1.4442.

Comment (caleb-james-smith) [reply python/modules/Stop0l_trigger.py]: Also, I just noticed that this
```(1.566 < abs(photon.eta) and abs(photon.eta) < 2.5)```
should be
```(1.566 > abs(photon.eta) and abs(photon.eta) < 2.5)```

Comment (caleb-james-smith) [reply python/modules/Stop0l_trigger.py]: Fixed: https://github.com/susy2015/NanoSUSY-tools/commit/824e119a132cfb1530b5772f60cdebc004727df3

**Comment by caleb-james-smith** [comment]
Ok, I have fixed the eta cut mistake.

**Comment by zhenbinwu** [comment]
What is the purpose of this PR? It seems to me only the value 1.442 changed to 1.4442 on the photon boundary. This is very minor change. Also, the value to be used depending on what condition Hui used in his trigger efficiency study. @wang-hui Do you use 1.442 or 1.4442? Either, the impact will be negligible.

**Comment by caleb-james-smith** [comment]
Yes, the main change is |eta| < 1.442 ---> |eta| < 1.4442.
Even though it is a small change, I would like the eta cuts to be identical to our photon selection.

**Comment by caleb-james-smith** [comment]
Another change changing tabs to spaces. In general spaces are better than tabs because they will look the same with different editors and settings, unlike tabs.

**Comment by zhenbinwu** [comment]
@wang-hui What is the eta range when you measure the photon trigger efficiency? We just need to follow your condition when applying it.

**Comment by wang-hui** [comment]
The photon eta cut in my trigger efficiency root files is |eta| < 1.442. I strictly followed our old photon selection
https://github.com/susy2015/NanoSUSY-tools/blob/dev_v2/python/modules/Stop0lObjectsProducer.py#L144
If we decide to follow my condition, then we can just close this PR. No need to change anything

**Comment by zhenbinwu** [comment]
Ok, I see. I don't see the need of this PR given the same condition used in the measurement. Closing this PR

**Comment by caleb-james-smith** [comment]
However, this means that our photon trigger efficiency will have a different condition than our photon selection, right?

**Pull request closed without merging**